plugins/woocommerce/tests/php/src/Internal/AbilitiesApi/AbilitiesApiIntegrationTest.php

```
@@ -487,7 +485,7 @@ function () use ( $ability_id_1, $ability_id_2 ) {
 
 		// Create REST request - use version-appropriate namespace.
 		$are_abilities_in_wp_core = $this->are_abilities_in_wp_core();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$are_abilities_in_wp_core'. (undefined)

(UnusedLocalVariable)